02 - assignments/git_assignment.md

```diff
@@ -0,0 +1,45 @@
+# Git Assignment
+
+In this assignment, you will be learning more about `git` and `GitHub` by working through the questions. As you answer the questions, you will practice your git skills by staging, commiting and pushing your answers (changes) along the way.
+
+1. Create a new repository called `git_assignment` under the `UofT-DSI` GitHub organization without a `README.md`
```

Comment:
This is not possible, we cannot grant learners direct access to the DSI org, that being said, we're in talks to implement GitHub classrooms or worst case scenario, we ask the participants to fork the repos into their own personal repo.